boards/xtensa/m5stickc_plus/m5stickc_plus_defconfig

```diff
@@ -20,3 +20,5 @@ CONFIG_GEN_IRQ_VECTOR_TABLE=n
 
 CONFIG_I2C=y
 CONFIG_CLOCK_CONTROL=y
+CONFIG_REGULATOR=y
+CONFIG_REGULATOR_AXP192=y
```

Comment:
not sure this should be here as you haven't declared the regulator in the main devicetree but only in the overlay? probably be best to keep it here but also add a "default" definition for the regulator, with everything turned off/set to sensible default settings, and then show how to override with settings appropriate for turning on backlight etc. in the LVGL sample overlay?